inc/classes/subscriber/third-party/plugins/class-smush-subscriber.php

```diff
@@ -1,6 +1,9 @@
 <?php
```

Comment:
@Screenfeed and @Tabrisrp Since we are touching this file, what do you think about move it to `inc/ThirdParty/Plugins/Smush/Subscriber.php` and renaming the class? We could also move any other smush files into this location. Thoughts?

Reply:
I agree

Reply:
I will use this architecture. However, there is a sub-folder in `Plugins` currently, for the `AMP` plugin: `inc/ThirdParty/Plugins/Optimization/AMP.php`. Maybe we should harmonize.

Reply:
Done.